python/mdvtools/mdvproject.py

```
@@ -2081,6 +2242,157 @@
+            if np.any(finite_mask):
+                finite_data = data[finite_mask]
+                col_info["minMax"] = [float(np.min(finite_data)), float(np.max(finite_data))]
+                
+                col_info["quantiles"] = {}
+                for q in [0.001, 0.01, 0.05]:
+                    col_info["quantiles"][str(q)] = [
+                        float(np.percentile(finite_data, 100 * q)),
+                        float(np.percentile(finite_data, 100 * (1 - q)))
+                    ]
+
+def get_column_info_polars(columns, dataframe: "pl.DataFrame | pl.LazyFrame", supplied_columns_only, num_rows: int):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)